driver/bpf/filler_helpers.h

```diff
@@ -56,6 +59,57 @@ static __always_inline struct file *bpf_fget(int fd)
 	return fil;
 }
 
+// Kernel 5.10 introduced a new bpf_helper called `bpf_d_path` to extract a file path starting from a file descriptor.
+// Libscap loads our bpf programs as `BPF_PROG_TYPE_RAW_TRACEPOINT` programs. This type of program doesn't seem able to call this new helper because it is out of its scope. For more details see here https://github.com/torvalds/linux/blob/58e1100fdc5990b0cc0d4beaf2562a92e621ac7d/kernel/trace/bpf_trace.c#L1574
+static __always_inline char *bpf_get_path(struct filler_data *data, int fd)
```

Comment:
This function looks good, but we might want to test it across multiple clang/bpf-verifier combinations to make sure that it does not break the probe on certain kernel versions.

Reply:
I have tested the bpf function for multiple clang/bpf-verifier combinations, see the PR description for more details.